src/http/graphql_data_loader.rs

```diff
@@ -4,10 +4,10 @@ use std::sync::Arc;
 use std::time::Duration;
 
 use async_graphql::async_trait;
-use async_graphql::dataloader::{DataLoader, Loader, NoCache};
 use async_graphql::futures_util::future::join_all;
 
 use crate::config::Batch;
+use crate::dataloader::{DataLoader, Loader, NoCache};
 use crate::http::{DataLoaderRequest, HttpClient, Response};
 
 pub struct GraphqlDataLoader {
```

Comment:
This file should be in `/graphql` folder. Can do it here or in a different PR.

Reply:
should be a small change. will do that in this PR itself